Automattic/jetpack
# Temporary fix for wp-env database issue

This is a temp solution for wp env database connection issue

#### Changes proposed in this Pull Request:

* Use latest wp-env from trunk. Solution found here: 84-gh-Automattic/a8c-wp-env
* Temporary solution until https://github.com/WordPress/gutenberg/pull/29800 gets released to npm

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
n/a

#### Testing instructions:
```
yarn install
yarn wp-env destroy
yarn wp-env start
```
WP env should start successfully.
